test/gerkin/tests/features/data_update.feature

```diff
@@ -0,0 +1,15 @@
+Feature: API endpoint allowing users to update a new record in the Digital Registries database.
+  Request endpoint: PUT /data/{registryname}/{versionnumber}/update
+
+  Background:
+    Given The user wants to update a record in the Digital Registries database
+
+  Scenario: The user successfully updates the record in the Digital Registries database
+    When The user triggers an action to update a record in the database
+    And The request with a valid payload is sent
+    Then The user receives a success message
```

Comment:
Is a success message enough here or do we want to verify that data was changed?

Reply:
We are not sure what messages exactly user will receive, so I think success will be enough